Kubernetes/windows/fluent-bit/fluent-bit.conf

```diff
@@ -0,0 +1,22 @@
+[SERVICE]
+    Flush        15
+    Daemon       Off
+    Log_Level    info
+    Log_File      /etc/fluent-bit/fluent-bit.log
+    HTTP_Server  Off
+    HTTP_Listen  0.0.0.0
+    HTTP_Port    2020
+
+[INPUT]
+    Name        forward
+    Listen      127.0.0.1
+    Port        25230
+    Chunk_Size  32
```

Comment:
Set Mem_Buf_limit to 10m

Reply:
https://github.com/microsoft/Docker-Provider/blob/ci_feature_prod/installer/conf/td-agent-bit.conf#L15

Reply:
Add task : These settings are configurable